solitude/settings/products.py

```diff
@@ -0,0 +1,20 @@
+# A definiton of Products for Payments for Firefox Accounts. Marketplace
+# defines these dynamically. However for the moment we will define these from
+# configuration files.
+from decimal import Decimal
+
+products = {
+    'concrete': {
+        'seller': 'mozilla-concrete',
+        'products': [
+            {
+                'name': 'brick',
+                'amount': Decimal('10')
```

Comment:
it seems a little weird to me that if you want to change the price of a product you'd have to do so in both braintree and the config file.